torch/csrc/jit/runtime/profiling_graph_executor_impl.cpp

```diff
@@ -313,11 +315,13 @@ void ProfilingGraphExecutorImpl::runProfilingOptimizations(
     runNoGradOptimizations(copy);
   }
   EliminateDeadCode(copy);
+  ClearProfilingInformation(copy);
```

Comment:
Why do we now need this?
Also, please add a `GRAPH_DUMP`statement before it.

Reply:
Yea, I think this locally before the rebase, it's not needed since the tensorexpr fuser will remove the profiles.